examples/repo-with-cluster-selectors-compiled-json-without-clusters-flag/cluster-dev/bookstore1/configmap_cm-all.json

```diff
@@ -10,6 +10,5 @@
 			}
 		}
 	],
-	"kind": "List",
-	"metadata": {}
```

Comment:
It seems that only the json output is affected by this PR. (The yaml output is not affected). Is this intended?

Should we add something in the release notes for this? Some customers use `nomos hydrate` to generate the config for each cluster, and will notice this.

Reply:
Extracted this code to https://github.com/GoogleContainerTools/kpt-config-sync/pull/81

In the other PR, only the metadata field has been removed, not the kind. I fixed the Kind by adding GVK to the UnstructuredList.

The reason YAML didn't change is because YAML supports multiple objects per file, so it's not being wrapped in a List object like the JSON printer is. These modified fields are just on the List object.